extensions/wikia/VideoHandlers/js/handlers/Ooyala.js

```diff
@@ -165,9 +166,9 @@ define('wikia.videohandler.ooyala', [
 				log('All Ooyala assets loaded', log.levels.info, logGroup);
 
 				win.OO.ready(function () {
-					if (sourcePoint.isEnabled()) {
-						sourcePoint.addOnBlockingCallback(initRecoveredPlayer);
-						sourcePoint.addOnNotBlockingCallback(initRegularPlayer);
+					if (adBlockDetection.isEnabled()) {
+						adBlockDetection.addOnBlockingCallback(initRecoveredPlayer);
```

Comment:
IMO it's still only SP not detection in general,  so the name of module isn't adequate